README.md

```diff
@@ -101,9 +101,7 @@ pip install -r requirements_nocuda.txt
 
 ```
 cd text-generation-webui
-mkdir repositories
```

Comment:
repositories is not guaranteed to exist

Reply:
Yep, check the last part of this line: `git clone https://github.com/turboderp/exllama repositories/exllama`

It will create `repositories/` if it doesn't already exist.  I thought of this and tested it out to work

Reply:
Nice